### PR TITLE
Prince/ added country utility

### PR DESCRIPTION
**Pull request by prince-deriv**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,5 +68,8 @@
         "vite": "^5.1.2",
         "vite-plugin-dts": "^3.7.2",
         "vitest": "^1.2.2"
+    },
+    "dependencies": {
+        "js-cookie": "^3.0.5"
```
Comment (niloofar-deriv) [inline package.json:73]: Could u pls move this to dev dependencies this should fix the error

Comment (niloofar-deriv) [reply package.json:73]: + installing this
`npm install --save-dev @types/js-cookie`

**Review by matin-deriv — COMMENTED** [review]
Update the utils doc as well.

File: src/utils/country.utils.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import Cookies from "js-cookie";
+import { cloudflareTrace } from "../constants/url.constants";
+
+interface TraceData {
```
Comment (niloofar-deriv) [inline src/utils/country.utils.ts:4]:
```suggestion
type TraceData = {
```

Just for consistency

File: src/utils/country.utils.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    loc?: string;
+}
+export const getCountry = async (): Promise<string> => {
```
Comment (niloofar-deriv) [inline src/utils/country.utils.ts:7]: Could u pls add JSDOCS for this function + usage examples 🙏 
Screenshot 2024-11-06 at 5 33 57 PM

File: src/utils/country.utils.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import Cookies from "js-cookie";
```
Comment (niloofar-deriv) [inline src/utils/country.utils.ts:1]: Please export the component here as well
`src/utils/index.ts `

**Review by matin-deriv — APPROVED** [review]

**Pull request merged**